advanced/load_save.rst

```diff
@@ -0,0 +1,247 @@
+.. _load_save:
+
+Game and settings load and save system
+======================================
+
+Escoria comes with a complete Save and Load manager that takes care of saving a
+player's game state so that it's possible to load a previously saved game.
+
+It also allows for the saving and loading of game settings. Both settings and
+save game files can optionally contain user-defined settings.
+
+Loading and saving games
+------------------------
+
+Escoria manages save games using numbered slots. A save game slot is required
+to be provided as part of the save game function call.  If the given slot
+already has a save game, that saved game will be overwritten. The
+:doc:`ESCSaveManager.load_game() </api/ESCSaveManager>` function can be used to
+give the player the opportunity to choose whether to overwrite the existing
+save game or not.
+
+There is no limitation on the number of save slots that can be used.
+
+Saving a game is as simple as calling the :doc:`ESCSaveManager.save_game()
+</api/ESCSaveManager>` function.
+
+Files are saved to the folder specified in the Godot's Project Settings under
+``escoria/main/game_version``. If not changed, the default is '<game
+folder>/saves'.
+
+Loading a game works the same way, using a slot number to load the
+relevant save game. To load a game, simply call
+:doc:`ESCSaveManager.load_game() </api/ESCSaveManager>` function.
+
+Attempting to load a saved game that doesn't exist will result in an error
+message being displayed ("Save file <id> doesn't exist").
+
+Files are loaded from the folder specified in the Godot's Project Settings
+under ``escoria/main/game_version``. If not changed, the default is '<game
```

Comment:
Same here. Game_version isn't the right setting.

Reply:
Submitted a suggested change for @StraToN to approve that fixes this.